### PR TITLE
PIE/BAR otherOption error due to unavailability and enterprise Icon

### PR DESCRIPTION
**Pull request by nxanil**
@ronakmshah @xzhao025

Please review the issue number:
#495 
#494 
#496 

I have configured a new property to OtherOptions i.e. "minimum" to override the minimum of 10 slices.

### CODE REVIEW

**Comment by nxanil** [comment]
#493 as well!

File: src/components/App/MainMenu.js [hunk 1]
```diff
@@ -90,9 +90,9 @@ class MainMenuView extends React.Component {
                             primaryText={domain.name}
                             style={style.nestedItem}
                             innerDivStyle={style.innerNestedItem}
-                            onTouchTap={() => { this.props.goTo(process.env.PUBLIC_URL + "/dashboards/" + targetedDashboard, queryParams)}}
+                            onTouchTap={() => { this.props.goTo(`${process.env.PUBLIC_URL}/dashboards/${targetedDashboard}`, queryParams)}}
```
Comment (ronakmshah) [inline src/components/App/MainMenu.js]: This part was working before... why to change?

File: src/components/App/MainMenu.js [hunk 2]
```diff
@@ -218,7 +218,13 @@ class MainMenuView extends React.Component {
                     <p>{visualizationType} Analytics</p>
                 </div>
 
-                <Subheader style={style.subHeader}>ENTERPRISES <span className="pull-right"><img style={style.iconSubMenu} src="/icons/icon-enterprise.png" alt="" /></span></Subheader>
+                <Subheader style={style.subHeader}>
+                    ENTERPRISES
+                    <span className="pull-right">
+                      <img style={style.iconSubMenu} src={`${process.env.PUBLIC_URL}/icons/icon-enterprise.png`} alt="" />
```
Comment (ronakmshah) [inline src/components/App/MainMenu.js]: Isnt this line the only change that needs to change?

Comment (nxanil) [reply src/components/App/MainMenu.js]: Agreed, and it is same as previous, but used the latest format as per ECMA 6 and have better readbility. A bit of refactoring along with fixing the issue.

For your reference: https://developers.google.com/web/updates/2015/01/ES6-Template-Strings

**Pull request merged**